lineman/app/app.coffee

```diff
@@ -69,8 +69,9 @@ angular.module('loomioApp').controller 'ApplicationController', ($scope, $filter
     {path: '/m/:key/votes/new', component: 'proposalRedirect' },
     {path: '/g/:key/memberships', component: 'membershipsPage'},
     {path: '/g/:key/membership_requests', component: 'membershipRequestsPage'},
+    {path: '/g/:key/previous_proposals', component: 'previousProposalsPage'}
```

Comment:
Hrrm.. we don't know if they were actually "decisions" given the way proposals are used..